### PR TITLE
Fixed issue#945 by adding Order to "filter by name"

### PR DESCRIPTION
**Pull request by juliankrieger**
Names are now ordered alphabetically.

### CODE REVIEW

File: _includes/scripts.html [hunk 1]
```diff
@@ -2,9 +2,9 @@
   <div class="filters-panel cf">
     <h5>Filter by name: </h5>
     <select class="names-filter" multiple data-placeholder="Select a project..." >
-    <% _.each(names, function(entry, key){ %>
-      <option value="<%-key%>"><%- entry.name%></option>
-    <% }) %>
+      <% _.each(_.sortBy(labels, function(entry, key){return entry.name;}), function(entry, key){ %>
```
Comment (juliankrieger) [inline _includes/scripts.html]: Added a js _ function here to sort the array alphabetifcally

**Comment by ritwik12** [comment]
@juliankrieger Can you show its working too. A hosted link or a screenshot?

**Comment by juliankrieger** [comment]
Yeah, give me a minute

**Comment by juliankrieger** [comment]
http://oi64.tinypic.com/2nai1ww.jpg @ritwik12

**Comment by juliankrieger** [comment]
@ritwik12  Any updates?

**Comment by ritwik12** [comment]
@juliankrieger Sorry for the delay, Busy nowadays.

**Comment by ritwik12** [comment]
Solves #945

**Comment by ritwik12** [comment]
@juliankrieger Can you also help with "Filter by label: " and "Filter by tags: " for the same?

**Pull request merged**